here-naksha-lib-psql/src/commonMain/kotlin/naksha/psql/executors/write/InstantWriteExecutor.kt

```diff
@@ -0,0 +1,45 @@
+package naksha.psql.executors.write
+
+import naksha.model.Naksha.NakshaCompanion.quoteIdent
+import naksha.model.Tuple
+import naksha.model.objects.NakshaFeature
+import naksha.psql.PgCollection
+import naksha.psql.PgColumn
+import naksha.psql.PgSession
+import naksha.psql.executors.write.WriteFeatureUtils.allColumnValues
+
+class InstantWriteExecutor(
+    val session: PgSession,
+): WriteExecutor {
+
+    override fun removeFeatureFromDel(collection: PgCollection, featureId: String) {
+        collection.deleted?.let { delTable ->
+            val quotedDelTable = quoteIdent(delTable.name)
+            val quotedIdColumn = quoteIdent(PgColumn.id.name)
+            session.usePgConnection()
+                .execute(
+                    sql = "DELETE FROM $quotedDelTable WHERE $quotedIdColumn='$featureId'"
+                )
```

Comment:
Possible SQL injections, please use a prepared statement with $1 instead of $featureId.

Reply:
true